helm/templates/agent-deployment.yaml

```
@@ -67,6 +67,8 @@ spec:
                   fieldPath: spec.nodeName
             - name: MY_NAMESPACE
               value: {{ include "nuvlaedge.namespace" . }}
+            - name: JOB_ENGINE_IMAGE_PULL_POLICY
+              value: {{ .Values._NE_JOB_ENGINE_PULL_POLICY | default "Always" | quote }}
```

On the Docker version, env vars related to the job executors starts with `JOB_` (for example `JOB_PULL_TIMEOUT`).
So for consistency between Docker Compose and Kubernetes/Helm installation, I would suggest to name it `JOB_IMAGE_PULL_POLICY` (both for the name of helm parameter and env var)

```suggestion
            - name: JOB_IMAGE_PULL_POLICY
              value: {{ .Values.JOB_IMAGE_PULL_POLICY | default "Always" | quote }}
```

---

@schaubl Yes, this works for me as well. Initially, I think, the idea was to "hide" this option, but in fact, I agree there is no real reason for that.